src/data/graphics/items.h

```diff
@@ -1981,7 +1993,6 @@ const u32 gItemIconPalette_Scanner[] = INCBIN_U32("graphics/items/icon_palettes/
 const u32 gItemIcon_StorageKey[] = INCBIN_U32("graphics/items/icons/storage_key.4bpp.lz");
 
 const u32 gItemIcon_KeyToRoom1[] = INCBIN_U32("graphics/items/icons/key_to_room_1.4bpp.lz");
-const u32 gItemIconPalette_Key[] = INCBIN_U32("graphics/items/icon_palettes/key.gbapal.lz");
```

Comment:
This should remain here as the intention is for the first item icon that uses a shared palette has it right underneath its icon declaration.